### ISSUE TITLE
Shortcuts for WET

### ISSUE THREAD

**Issue by milupo** [Lusito/web-ext-translator#2]
Hi Lusito,

are there any shortcuts for WET? I'm especially interested in a shortcut that switches to the next string saving the previous string.

Thanks.
Michael

**Comment by Lusito** [comment]
Normal keyboard navigation shortcuts exist like in any other browser.
In this case, pressing tab a couple of times would do the trick (and shift + tab for the reverse direction).
Other than that, I'm afraid there is nothing like it at the moment.
It shouldn't be too hard to create something if tab does not suffice, but I'm currently busy moving to a new job and city.

**Comment by milupo** [comment]
Ok. It's a pity. I know that I can use the tab key a couple of times but it is somewhat inconvenient. I thought of "Strg+Enter" as on Pontoon. Nevertheless, thank you for creating WET.

**Comment by Lusito** [comment]
Just for reference, when I get to it:
Ctrl+Enter = next line, but same language?

**Comment by milupo** [comment]
Yes, it should go to the next string and the just translated string should be saved at the time. For now, I have to click in the next string and I must not forget to save the translated strings that I don't loose them by mischance.

**Comment by Lusito** [comment]
Forgot to mention: These shortcuts exist in addition to normal browser keyboard shortcuts:
ctrl + z / y (undo/redo)
ctrl + s (save)

What do you mean by save? Save to files? => press ctrl + s

**Comment by Lusito** [comment]
There is no way to just save one language or even just one string. At the moment when you press save, all languages are persisted in the json files.

**Comment by milupo** [comment]
OK. I hoped that there is a more comfortable way. It would make the translation process smoother. One shortcut for two actions at the same time.

**Comment by Lusito** [comment]
I'll consider it for future versions.
